vercel/speed-insights
# Upgrade example to Svelte 5 / SvelteKit 2 (Fixes #77)

### 📓 What's in there?

Installation is currently broken on a fresh Next.js repo because speed-analytics relies on an outdated version of Svelte.

### 🧪 How to test?

- Run pnpm dev in apps/sveltekit
  - Observe it works as before

-->
